src/libs/Navigation/getTopmostCentralPaneName.ts

```diff
@@ -0,0 +1,19 @@
+import type {NavigationState, PartialState} from '@react-navigation/native';
+import SCREENS from '@src/SCREENS';
+
+// Get the name of topmost report in the navigation stack.
+function getTopmostCentralPaneName(state: NavigationState | PartialState<NavigationState>): string | undefined {
```

Comment:
```suggestion
function getTopmostSettingsCentralPaneName(state: NavigationState | PartialState<NavigationState>): string | undefined {
```

The function name is a little confusing since we have two central panes, the old one that lives along the bottom tab navigator and the new one that comes with the full screen navigator (the one used for settings).